commit a32c864292e147b769bc6f2b19735593cf84f41e
Author: Jehan Monnier <jehan.monnier@linphone.org>
Date:   Mon Apr 14 16:28:57 2014 +0200

    fix invalid read in gtk app call log management

diff --git a/gtk/calllogs.c b/gtk/calllogs.c
index 2f7e8f7..7a4840a 100644
--- a/gtk/calllogs.c
+++ b/gtk/calllogs.c
@@ -293,7 +293,10 @@ void linphone_gtk_call_log_update(GtkWidget *w){
 #endif
 		lf=linphone_core_get_friend_by_address(linphone_gtk_get_core(),addr);
 		if(lf != NULL){
-			display=linphone_address_get_display_name(linphone_friend_get_address(lf));
+			if ((display=linphone_address_get_display_name(linphone_friend_get_address(lf)))) {
+				/*update display name from friend*/
+				linphone_address_set_display_name(la,display);
+			}
 		} else {
 			display=linphone_address_get_display_name(la);
 		}
@@ -302,9 +305,8 @@ void linphone_gtk_call_log_update(GtkWidget *w){
 			if (display==NULL){
 				display=linphone_address_get_domain (la);
 			}
-		} else {
-			linphone_address_set_display_name(la,display);
 		}
+
 		if (linphone_call_log_get_quality(cl)!=-1){
 			snprintf(quality,sizeof(quality),"%.1f",linphone_call_log_get_quality(cl));
 		}else snprintf(quality,sizeof(quality)-1,"%s",_("n/a"));
diff --git a/tester/call_tester.c b/tester/call_tester.c
index 2a18b30..c655e4b 100644
--- a/tester/call_tester.c
+++ b/tester/call_tester.c
@@ -355,7 +355,7 @@ static void call_with_dns_time_out(void) {
 	linphone_core_set_sip_transports(marie->lc,&transport);
 	linphone_core_iterate(marie->lc);
 	sal_set_dns_timeout(marie->lc->sal,0);
-	linphone_core_invite(marie->lc,"sip:toto@toto.com");
+	linphone_core_invite(marie->lc,"\"t\x8et\x8e\" sip:toto@toto.com"); /*just to use non ascii values*/
 	for(i=0;i<10;i++){
 		ms_usleep(200000);
 		linphone_core_iterate(marie->lc);
