--- linphone-1.2.0/share/linphone.desktop~	2006-01-23 08:43:33.000000000 +0100
+++ linphone-1.2.0/share/linphone.desktop	2006-01-23 08:43:33.000000000 +0100
@@ -1,7 +1,9 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Linphone
 Comment=Linphone is a web-phone
-Comment[fr]=Linphone est un web-phone.
-Comment[de]=Linphone ist ein web-phone.
+Comment[fr]=Linphone est un web-phone
+Comment[de]=Linphone ist ein web-phone
+Comment[pl]=Telefon internetowy Linphone
 Type=Application
 Exec=linphone
--- linphone-1.2.0/share/linphone_applet.desktop~	2006-01-23 08:43:33.000000000 +0100
+++ linphone-1.2.0/share/linphone_applet.desktop	2006-01-23 08:43:33.000000000 +0100
@@ -1,8 +1,10 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Linphone Applet
 Comment=Linphone is a web-phone
-Comment[fr]=Linphone est un web-phone.
-Comment[de]=Linphone ist ein web-phone.
+Comment[fr]=Linphone est un web-phone
+Comment[de]=Linphone ist ein web-phone
+Comment[pl]=Telefon internetowy Linphone
 Type=Application
 Exec=linphone --applet --activate-goad-server=linphone_applet
 Icon=linphone/linphone2.png
