diff -urNp -x '*.orig' linphone-3.12.0.org/Makefile.am linphone-3.12.0/Makefile.am
--- linphone-3.12.0.org/Makefile.am	2017-04-06 11:25:59.000000000 +0200
+++ linphone-3.12.0/Makefile.am	2021-07-06 00:14:06.742751269 +0200
@@ -4,7 +4,7 @@
 ACLOCAL_AMFLAGS = -I m4 $(ACLOCAL_MACOS_FLAGS)
 
 SUBDIRS = build m4 pixmaps po @ORTP_DIR@ @MS2_DIR@ \
-		coreapi console gtk share scripts tools daemon tester include
+		coreapi console gtk share scripts tools daemon tester include wrappers/cpp
 
 GITVERSION=`cd $(top_srcdir) && git describe --always || echo $(VERSION)`
 
diff -urNp -x '*.orig' linphone-3.12.0.org/configure.ac linphone-3.12.0/configure.ac
--- linphone-3.12.0.org/configure.ac	2021-07-06 00:14:06.539414973 +0200
+++ linphone-3.12.0/configure.ac	2021-07-06 00:14:06.742751269 +0200
@@ -1073,6 +1073,12 @@ AC_CONFIG_FILES([
 	include/Makefile
 	include/linphone/Makefile
 	coreapi/Makefile
+	coreapi/help/Makefile
+	coreapi/help/doc/Makefile
+	coreapi/help/doc/doxygen/Doxyfile
+	coreapi/help/doc/doxygen/Makefile
+	coreapi/help/examples/Makefile
+	coreapi/help/examples/C/Makefile
 	tester/Makefile
 	gtk/Makefile
 	console/Makefile
@@ -1089,6 +1095,8 @@ AC_CONFIG_FILES([
 	share/audio-assistant.desktop
 	scripts/Makefile
 	tools/Makefile
+	wrappers/cpp/Makefile
+	wrappers/cpp/linphone++.pc
 	linphone.spec
 	linphone.iss
 ])
diff -urNp -x '*.orig' linphone-3.12.0.org/coreapi/Makefile.am linphone-3.12.0/coreapi/Makefile.am
--- linphone-3.12.0.org/coreapi/Makefile.am	2017-06-20 12:03:59.000000000 +0200
+++ linphone-3.12.0/coreapi/Makefile.am	2021-07-06 00:14:06.742751269 +0200
@@ -13,7 +13,7 @@ GITLOG=`cd $(top_srcdir) && git log -1 -
 
 ECHO=/bin/echo
 
-SUBDIRS=.
+SUBDIRS=. help
 
 EXTRA_DIST=linphonecore_jni.cc $(GITVERSION_FILE)
 
diff -urNp -x '*.orig' linphone-3.12.0.org/coreapi/help/doc/doxygen/Makefile.am linphone-3.12.0/coreapi/help/doc/doxygen/Makefile.am
--- linphone-3.12.0.org/coreapi/help/doc/doxygen/Makefile.am	2017-06-20 12:03:59.000000000 +0200
+++ linphone-3.12.0/coreapi/help/doc/doxygen/Makefile.am	2021-07-06 00:14:06.742751269 +0200
@@ -1,6 +1,6 @@
 EXTRA_DIST=Doxyfile.in doxygen.dox
 
-SOURCES=doxygen.dox $(top_srcdir)/coreapi/help/*.c $(top_srcdir)/coreapi/*.c $(top_srcdir)/coreapi/*.h
+SOURCES=doxygen.dox $(top_srcdir)/coreapi/*.c $(top_srcdir)/coreapi/*.h
 
 if HAVE_DOXYGEN
 
@@ -9,21 +9,21 @@ pkgdocdir=$(docdir)/$(PACKAGE)-$(VERSION
 doc_htmldir=$(pkgdocdir)/html
 doc_xmldir=$(pkgdocdir)/xml
 
-doc_html_DATA = $(top_builddir)/coreapi/help/doc/html/html.tar
+doc_html_DATA = $(top_builddir)/coreapi/help/doc/doxygen/html/html.tar
 
-$(doc_html_DATA): $(top_builddir)/coreapi/help/doc/html/index.html
-	cd $(top_builddir)/coreapi/help/doc/html/ && tar cf html.tar *
+$(doc_html_DATA): $(top_builddir)/coreapi/help/doc/doxygen/html/index.html
+	cd $(top_builddir)/coreapi/help/doc/doxygen/html/ && tar cf html.tar *
 
-$(top_builddir)/coreapi/help/doc/html/index.html: $(SOURCES) Doxyfile Makefile.am
+$(top_builddir)/coreapi/help/doc/doxygen/html/index.html: $(SOURCES) Doxyfile Makefile.am
 	rm -rf doc
 	$(DOXYGEN) Doxyfile
 
-doc_xml_DATA = $(top_builddir)/coreapi/help/doc/xml/xml.tar
+doc_xml_DATA = $(top_builddir)/coreapi/help/doc/doxygen/xml/xml.tar
 
-$(doc_xml_DATA): $(top_builddir)/coreapi/help/doc/xml/index.xml
-	cd $(top_builddir)/coreapi/help/doc/xml/ && tar cf xml.tar *
+$(doc_xml_DATA): $(top_builddir)/coreapi/help/doc/doxygen/xml/index.xml
+	cd $(top_builddir)/coreapi/help/doc/doxygen/xml/ && tar cf xml.tar *
 
-$(top_builddir)/coreapi/help/doc/xml/index.xml: $(top_builddir)/coreapi/help/doc/html/index.html
+$(top_builddir)/coreapi/help/doc/doxygen/xml/index.xml: $(top_builddir)/coreapi/help/doc/doxygen/html/index.html
 
 
 install-data-hook:
diff -urNp -x '*.orig' linphone-3.12.0.org/wrappers/cpp/Makefile.am linphone-3.12.0/wrappers/cpp/Makefile.am
--- linphone-3.12.0.org/wrappers/cpp/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ linphone-3.12.0/wrappers/cpp/Makefile.am	2021-07-06 00:14:06.742751269 +0200
@@ -0,0 +1,20 @@
+lib_LTLIBRARIES = liblinphone++.la
+liblinphone___la_CPPFLAGS = -DLINPHONECXX_EXPORTS
+liblinphone___la_CXXFLAGS = -I$(builddir)/include -I$(top_builddir)/include -I$(top_srcdir)/include $(BCTOOLBOX_CFLAGS) $(BELLESIP_CFLAGS)
+liblinphone___la_SOURCES = object.cc tools.cc
+nodist_liblinphone___la_SOURCES = src/linphone++.cc
+liblinphone___la_LDFLAGS= -version-info $(LIBLINPHONE_SO_VERSION) -no-undefined
+liblinphone___la_LIBADD = $(top_builddir)/coreapi/liblinphone.la $(BCTOOLBOX_LIBS) $(BELLESIP_LIBS)
+
+linphonexxincludedir = $(includedir)/linphone++
+linphonexxinclude_HEADERS = object.hh
+
+pkgconfigdir = $(libdir)/pkgconfig
+nodist_pkgconfig_DATA = linphone++.pc
+
+src/linphone++.cc include/linphone++/linphone.hh: $(top_srcdir)/tools/genapixml.py $(top_srcdir)/tools/metadoc.py $(top_srcdir)/tools/abstractapi.py genwrapper.py class_header.mustache class_impl.mustache enums_header.mustache main_header.mustache $(top_builddir)/coreapi/help/doc/doxygen/xml/index.xml
+	$(PYTHON) $(srcdir)/genwrapper.py $(top_builddir)/coreapi/help/doc/doxygen/xml
+
+install-data-hook:
+	install -d $(DESTDIR)$(linphonexxincludedir)
+	cp -p $(builddir)/include/linphone++/*.hh $(DESTDIR)$(linphonexxincludedir)
diff -urNp -x '*.orig' linphone-3.12.0.org/wrappers/cpp/linphone++.pc.in linphone-3.12.0/wrappers/cpp/linphone++.pc.in
--- linphone-3.12.0.org/wrappers/cpp/linphone++.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ linphone-3.12.0/wrappers/cpp/linphone++.pc.in	2021-07-06 00:14:06.742751269 +0200
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Requires.private: linphone bctoolbox belle-sip
+Name: liblinphone++
+Description: C++ wrapper for linphone libraries.
+Version: @VERSION@
+Libs: -llinphone++
+Cflags: -I$(includedir)
