--- linphone-0.11.0/mediastreamer/Makefile.am.orig	Mon May  5 16:30:01 2003
+++ linphone-0.11.0/mediastreamer/Makefile.am	Fri Jun 27 12:12:12 2003
@@ -72,8 +72,8 @@
 															
 					
 libmediastreamer_la_LIBADD= $(GLIB_LIBS) \
-							../gsmlib/libgsm.la \
-							../lpc10-1.5/liblpc10.la \
+							-lgsm \
+							-llpc10 \
 							../oRTP/src/libortp.la
 	
 #the xine wrapper video output */
@@ -89,7 +89,7 @@
 #the ffmpeg wrapper video encoder and decoder
 libmsffmpeg_la_SOURCES=msavencoder.c msavencoder.h\
 						msavdecoder.c msavdecoder.h
-libmsffmpeg_la_LIBADD=libmediastreamer.la ../ffmpeg/libavcodec/libavcodec.la
+libmsffmpeg_la_LIBADD=libmediastreamer.la -lavcodec
 
 #the gdk video output plugin
 libmsgdkvo_la_SOURCES=msvideooutput.c msvideooutput.h
@@ -98,7 +98,7 @@
 
 #the speex encoder and decoder 
 libmsspeex_la_SOURCES=msspeexenc.c msspeexenc.h msspeexdec.c msspeexdec.h
-libmsspeex_la_LIBADD=libmediastreamer.la $(top_srcdir)/speex/libspeex/libspeex.la
+libmsspeex_la_LIBADD=libmediastreamer.la -lspeex
 
 #the v4l plugin 
 libmsv4l_la_SOURCES=msv4l.c msv4l.h
@@ -178,9 +178,6 @@
 						-I$(top_srcdir)/mediastreamer \
 						-I$(top_srcdir)/oRTP/src \
 						-I$(top_srcdir)/oRTP \
-						-I$(top_srcdir)/gsmlib   \
-						-I$(top_srcdir)/lpc10-1.5    \
-                        -I$(top_srcdir)/ffmpeg/libavcodec \
+                        -I/usr/include/ffmpeg \
 						-I/usr/X11R6/include/X11 \
-						-I/usr/include/SDL \
-						-I$(top_srcdir)/speex/libspeex
+						-I/usr/include/SDL
--- linphone-0.11.0/media_api/Makefile.am.orig	Mon Sep 23 14:13:20 2002
+++ linphone-0.11.0/media_api/Makefile.am	Fri Jun 27 12:10:21 2003
@@ -22,11 +22,7 @@
 DEFS=@DEFS@ @SOUNDDEFS@ -DDEBUG -DG_LOG_DOMAIN=\"MediaApi\" 
 
 INCLUDES=-I$(top_srcdir)/mediastreamer \
-		-I$(top_srcdir)/speex \
-		-I$(top_srcdir)/gsmlib \
 		-I$(top_srcdir)/oRTP \
-		-I$(top_srcdir)/oRTP/src \
-		-I$(top_srcdir)/lpc10-1.5 \
-		-I$(top_srcdir)/ffmpeg 
+		-I$(top_srcdir)/oRTP/src
 
 
--- linphone-0.11.0/coreapi/Makefile.am.orig	Thu Mar 27 11:31:16 2003
+++ linphone-0.11.0/coreapi/Makefile.am	Fri Jun 27 12:08:45 2003
@@ -6,7 +6,6 @@
 	-I$(top_srcdir)/oRTP/src \
 	-I$(top_srcdir)/oRTP/ \
 	-I$(top_srcdir)/mediastreamer \
-	-I$(top_srcdir)/speex/libspeex \
 	-I$(osip_prefix)/include/ 
 
 EXTRA_DIST= parse-path.cP 
--- linphone-0.11.0/console/Makefile.am.orig	Fri Feb 28 15:48:37 2003
+++ linphone-0.11.0/console/Makefile.am	Fri Jun 27 12:08:55 2003
@@ -7,7 +7,6 @@
 	-I$(top_srcdir)/oRTP/src \
 	-I$(top_srcdir)/oRTP/ \
 	-I$(top_srcdir)/mediastreamer \
-	-I$(top_srcdir)/speex/libspeex \
 	-I$(osip_prefix)/include/ 
 
 
--- linphone-0.11.0/gnome/Makefile.am.orig	Thu Mar 27 11:31:09 2003
+++ linphone-0.11.0/gnome/Makefile.am	Fri Jun 27 12:09:31 2003
@@ -8,7 +8,6 @@
 	-I$(top_srcdir)/oRTP/src \
 	-I$(top_srcdir)/oRTP/ \
 	-I$(top_srcdir)/mediastreamer \
-	-I$(top_srcdir)/speex/libspeex \
 	-I$(top_srcdir)/coreapi \
 	-I$(top_srcdir)/osipua/src 
 
--- linphone-0.11.0/Makefile.am.orig	Wed Mar 12 18:22:51 2003
+++ linphone-0.11.0/Makefile.am	Fri Jun 27 11:54:50 2003
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = pixmaps ffmpeg po macros debian ipkg oRTP gsmlib lpc10-1.5 speex mediastreamer \
+SUBDIRS = pixmaps po macros debian ipkg oRTP mediastreamer \
 		media_api osipua coreapi console gnome share developer-docs 
 
 EXTRA_DIST = config.rpath mkinstalldirs  linphone2.glade BUGS linphone.kdevprj \
--- linphone-0.11.0/configure.in.orig	Mon May 19 10:43:31 2003
+++ linphone-0.11.0/configure.in	Fri Jun 27 11:57:03 2003
@@ -178,24 +178,12 @@
 
 if test "$video" = "true"; then
 	echo "WARNING: Video support is under developpement, nothing works."
-	dnl check if nasm is present (needed to compile ffmpeg )
-	AC_CHECK_PROG(NASM, nasm, true, false)
-	
-	if $NASM ;then
-		cd $(top_builddir2)/ffmpeg; ln -sf Makefile.orig Makefile; cd ..
-		cd $(top_builddir2)/ffmpeg/libavcodec; ln -sf Makefile.libtool Makefile; cd .. ;cd ..
-		echo "Video support will be build, although it is not functionnal for the moment."	
-		AVCODEC_LIBS="../ffmpeg/libavcodec/libavcodec.la"
+		AVCODEC_LIBS="-lavcodec"
 		AC_SUBST(AVCODEC_LIBS)
 		AC_DEFINE(HAVE_AVCODEC,1,[Defines if we can use libavcodec library])
-	else
-		AC_MSG_ERROR("nasm is missing ! You require nasm to build the video support in linphone.")
-	fi
 	PKG_CHECK_MODULES(GDK,gdk-2.0 >= 2.0.0)
 	AC_SUBST(GDK_CFLAGS)
 	AC_SUBST(GDK_LIBS)
-else
-   cd $(top_builddir2)/ffmpeg; ln -sf Makefile.null Makefile ; cd ..
 fi
 
 dnl mediastreamer configuration
@@ -227,7 +215,7 @@
 AC_SUBST(SOUNDDEFS)
 
 
-AC_CONFIG_SUBDIRS( ffmpeg speex oRTP osipua)
+AC_CONFIG_SUBDIRS( oRTP osipua)
 
 ##################################################
 # Check for gtk-doc.
@@ -252,8 +240,6 @@
 mediastreamer/Makefile
 media_api/Makefile
 osipua/Makefile
-gsmlib/Makefile
-lpc10-1.5/Makefile
 coreapi/Makefile
 gnome/Makefile
 console/Makefile
