diff -dur -x '*~' linphone-3.4.3.orig/mediastreamer2/configure.ac linphone-3.4.3/mediastreamer2/configure.ac
--- linphone-3.4.3.orig/mediastreamer2/configure.ac	2011-03-28 15:04:21.000000000 +0200
+++ linphone-3.4.3/mediastreamer2/configure.ac	2011-04-30 13:45:13.000000000 +0200
@@ -586,7 +586,7 @@
 
 AC_SUBST(PACKAGE_PLUGINS_DIR)
 
-PACKAGE_DATA_DIR="$prefix/share"
+PACKAGE_DATA_DIR="$(datadir)/linphone"
 AC_DEFINE_UNQUOTED(PACKAGE_DATA_DIR,  "${package_prefix}/share" ,[path of data])
 AC_SUBST(PACKAGE_DATA_DIR)
 
diff -dur -x '*~' linphone-3.4.3.orig/mediastreamer2/src/Makefile.am linphone-3.4.3/mediastreamer2/src/Makefile.am
--- linphone-3.4.3.orig/mediastreamer2/src/Makefile.am	2011-02-09 09:43:11.000000000 +0100
+++ linphone-3.4.3/mediastreamer2/src/Makefile.am	2011-04-30 13:44:49.000000000 +0200
@@ -220,7 +220,7 @@
 
 AM_OBJCFLAGS=$(AM_CFLAGS)
 
-imgdir=$(datadir)/images/
+imgdir=$(datadir)/linphone/images/
 
 img_DATA=nowebcamCIF.jpg
 
