--- linphone-desktop-5.0.15/CMakeLists.txt.orig	2023-04-19 21:14:47.137607063 +0200
+++ linphone-desktop-5.0.15/CMakeLists.txt	2023-04-19 21:17:02.070209404 +0200
@@ -44,7 +44,7 @@ if( APPLE )
 	endif()
 endif()
 
-project(linphoneqt)
+project(linphoneqt C CXX)
 
 include(GNUInstallDirs)
 include(CheckCXXCompilerFlag)
@@ -250,8 +250,8 @@ if ( NOT Qt5_FOUND )
 	message(FATAL_ERROR "Minimum supported Qt5 version is 5.10!")
 endif()
 
-find_package(LinphoneCxx CONFIG QUIET)
 find_package(Linphone CONFIG QUIET)
+find_package(LinphoneCxx CONFIG QUIET)
 find_package(bctoolbox CONFIG QUIET)
 find_package(belcard CONFIG QUIET)
 find_package(Mediastreamer2 CONFIG QUIET)
--- linphone-desktop-4.2.5/linphone-app/CMakeLists.txt.orig	2021-07-17 17:21:38.970225216 +0200
+++ linphone-desktop-4.2.5/linphone-app/CMakeLists.txt	2021-07-17 20:21:15.985174448 +0200
@@ -66,8 +66,8 @@
 	endforeach( OUTPUTCONFIG CMAKE_CONFIGURATION_TYPES )
 endif()
 
-find_package(LinphoneCxx CONFIG)
 find_package(Linphone CONFIG)
+find_package(LinphoneCxx CONFIG)
 find_package(belcard CONFIG)
 find_package(Mediastreamer2 CONFIG)
 find_package(ortp CONFIG)
