diff -dur linphone-3.4.3.orig/gtk/main.c linphone-3.4.3/gtk/main.c
--- linphone-3.4.3.orig/gtk/main.c	2011-03-25 14:40:44.000000000 +0100
+++ linphone-3.4.3/gtk/main.c	2011-04-30 13:26:33.000000000 +0200
@@ -1431,8 +1431,9 @@
 	g_object_set(settings, "gtk-menu-images", TRUE, NULL);
 	g_object_set(settings, "gtk-button-images", TRUE, NULL);
 
-	if (workingdir!=NULL)
-		chdir(workingdir);
+	if (workingdir!=NULL && chdir(workingdir)) {
+		g_warning("Cannot chdir to the working directory: %s", workingdir);
+	}
 
 	/* Now, look for the factory configuration file, we do it this late
 		 since we want to have had time to change directory and to parse
