--- linphone-desktop-5.2.2/linphone-app/CMakeLists.txt.orig	2024-03-11 18:30:37.000000000 +0100
+++ linphone-desktop-5.2.2/linphone-app/CMakeLists.txt	2024-03-12 21:45:48.707479838 +0100
@@ -24,7 +24,7 @@ cmake_minimum_required(VERSION 3.22)
 
 
 #Linphone targets
-set(LINPHONE_PACKAGES LinphoneCxx Mediastreamer2 Belcard LibLinphone)
+set(LINPHONE_PACKAGES LinphoneCxx Mediastreamer2 BelCard LibLinphone)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
@@ -105,10 +105,7 @@ set(ENABLE_DB_STORAGE ON CACHE BOOLEAN "
 
 foreach(PACKAGE ${LINPHONE_PACKAGES})
 	message(STATUS "Trying to find ${PACKAGE}")
-	find_package(${PACKAGE})
-	if(NOT ${PACKAGE}_FOUND)
-		find_package(${PACKAGE} CONFIG REQUIRED)
-	endif()
+	find_package(${PACKAGE} CONFIG REQUIRED)
 endforeach()
 
 set(PLUGIN_TARGETS ${LinphoneCxx_TARGET})
@@ -112,7 +112,7 @@ set(PLUGIN_TARGETS ${LinphoneCxx_TARGET}
 set(APP_TARGETS ${LinphoneCxx_TARGET}
 				${BCToolbox_TARGET}#Logger/App
 				${Mediastreamer2_TARGET}#MediastreamerUtils
-				${Belcard_TARGET}#VCard Model
+				${BelCard_TARGET}#VCard Model
 				${LibLinphone_TARGET})#MediastreamerUtils
 
 ####################################
