diff -Naur linphone-0.11.0/share/C/Makefile.am linphone-0.11.0-p/share/C/Makefile.am
--- linphone-0.11.0/share/C/Makefile.am	Wed Dec 11 18:16:39 2002
+++ linphone-0.11.0-p/share/C/Makefile.am	Mon Jun 23 11:08:36 2003
@@ -13,9 +13,9 @@
 	cd manual; ln -sf t1.html index.html
 
 install-data-local:
-			$(mkinstalldirs) $(linphone_manualdir)
-			cp -rf $(linphone_help) $(linphone_manualdir)/.
+			$(mkinstalldirs) $(DESTDIR)$(linphone_manualdir)
+			cp -rf $(linphone_help) $(DESTDIR)$(linphone_manualdir)/.
 
 uninstall-local:
-			rm -rf $(linphone_manualdir)
+			rm -rf $(DESTDIR)$(linphone_manualdir)
 
diff -Naur linphone-0.11.0/share/fr/Makefile.am linphone-0.11.0-p/share/fr/Makefile.am
--- linphone-0.11.0/share/fr/Makefile.am	Wed Nov 28 23:27:47 2001
+++ linphone-0.11.0-p/share/fr/Makefile.am	Mon Jun 23 11:08:36 2003
@@ -11,9 +11,9 @@
 	cd manual; ln -sf t1.html index.html
 
 install-data-local:
-			$(mkinstalldirs) $(linphone_manualdir)
-			cp -rf $(linphone_help) $(linphone_manualdir)/.
+			$(mkinstalldirs) $(DESTDIR)$(linphone_manualdir)
+			cp -rf $(linphone_help) $(DESTDIR)$(linphone_manualdir)/.
 
 uninstall-local:
-			rm -rf $(linphone_manualdir)
+			rm -rf $(DESTDIR)$(linphone_manualdir)
 
diff -Naur linphone-0.11.0/share/it/Makefile.am linphone-0.11.0-p/share/it/Makefile.am
--- linphone-0.11.0/share/it/Makefile.am	Thu Jan 17 08:31:31 2002
+++ linphone-0.11.0-p/share/it/Makefile.am	Mon Jun 23 11:08:36 2003
@@ -11,9 +11,9 @@
 	cd manual; ln -sf t1.html index.html
 
 install-data-local:
-			$(mkinstalldirs) $(linphone_manualdir)
-			cp -rf $(linphone_help) $(linphone_manualdir)/.
+			$(mkinstalldirs) $(DESTDIR)$(linphone_manualdir)
+			cp -rf $(linphone_help) $(DESTDIR)$(linphone_manualdir)/.
 
 uninstall-local:
-			rm -rf $(linphone_manualdir)
+			rm -rf $(DESTDIR)$(linphone_manualdir)
 
diff -Naur linphone-0.11.0/share/ja/Makefile.am linphone-0.11.0-p/share/ja/Makefile.am
--- linphone-0.11.0/share/ja/Makefile.am	Fri Jan 10 22:57:06 2003
+++ linphone-0.11.0-p/share/ja/Makefile.am	Mon Jun 23 11:27:20 2003
@@ -11,9 +11,9 @@
 	cd manual; ln -sf t1.html index.html
 
 install-data-local:
-			$(mkinstalldirs) $(linphone_manualdir)
-			cp -rf $(linphone_help) $(linphone_manualdir)/.
+			$(mkinstalldirs) $(DESTDIR)$(linphone_manualdir)
+			cp -rf $(linphone_help) $(DESTDIR)$(linphone_manualdir)/.
 
 uninstall-local:
-			rm -rf $(linphone_manualdir)
+			rm -rf $(DESTDIR)$(linphone_manualdir)
 
