--- linphone-3.6.0/configure.ac.orig	2013-05-30 10:08:31.000000000 +0200
+++ linphone-3.6.0/configure.ac	2013-06-21 21:43:50.405687603 +0200
@@ -673,7 +673,7 @@
 AM_CONDITIONAL(BUILD_MSG_STORAGE, test x$enable_msg_storage = xtrue)
 if test x$enable_msg_storage != xfalse; then
 	PKG_CHECK_MODULES(SQLITE3,[ sqlite3 >= 3.7.0],[
-		SQLITE3_CFLAGS+="-DMSG_STORAGE_ENABLED"
+		SQLITE3_CFLAGS="$SQLITE3_CFLAGS -DMSG_STORAGE_ENABLED"
 		AC_SUBST(SQLITE3_CFLAGS)
 		AC_SUBST(SQLITE3_LIBS)
 		enable_msg_storage=true
