--- linphone-1.3.5/m4/exosip.m4.wiget	2005-02-04 16:19:59.000000000 +0100
+++ linphone-1.3.5/m4/exosip.m4	2006-05-04 23:43:42.000000000 +0200
@@ -30,8 +30,8 @@
 esac
 
 AC_CHECK_LIB(posix4,sem_open,[EXOSIP_LIBS="$EXOSIP_LIBS -lposix4 -mt"])
-AC_CHECK_LIB(nsl,nis_add,[EXOSIP_LIBS="$EXOSIP_LIBS -lnsl"])
-AC_CHECK_LIB(socket,sendto,[EXOSIP_LIBS="$EXOSIP_LIBS -lsocket"])
+dnl AC_CHECK_LIB(nsl,nis_add,[EXOSIP_LIBS="$EXOSIP_LIBS -lnsl"])
+AC_CHECK_FUNC(sendto,,[AC_CHECK_LIB(socket,sendto,[EXOSIP_LIBS="$EXOSIP_LIBS -lsocket"])])
 AC_CHECK_LIB(rt,clock_gettime,[EXOSIP_LIBS="$EXOSIP_LIBS -lrt"])
 dnl Checks for header files.
 AC_HEADER_STDC
--- linphone-1.3.5/mediastreamer2/Makefile.am.wiget	2006-03-28 16:54:20.000000000 +0200
+++ linphone-1.3.5/mediastreamer2/Makefile.am	2006-05-04 23:48:14.000000000 +0200
@@ -52,11 +52,11 @@
 	awk 'BEGIN { print("MSFilterDesc * ms_filter_descs[]={") } { printf("&%s,\n",$$1) } END{ print("NULL\n};\n") } ' filterdescs.txt >> $@
 	rm -f filterdescs.txt
 
-libmediastreamer2_la_LIBADD=	$(top_builddir)/oRTP/src/libortp.la \
+libmediastreamer2_la_LIBADD=	-lortp \
 				$(ALSA_LIBS) \
 				$(ARTS_LIBS) \
 				$(SPEEX_LIBS) \
-				$(top_builddir)/gsmlib/libgsm.la \
+				-lgsm \
 				-lrt
 
 if BUILD_VIDEO
@@ -65,8 +65,8 @@
 
 libquickstream_la_LIBADD=libmediastreamer2.la
 
-INCLUDES=-I$(top_srcdir)/oRTP/include -I$(top_srcdir)/gsmlib
+INCLUDES=-I/usr/include/ortp
 
-ORTP_CFLAGS=`cat $(top_builddir)/oRTP/ortp.defs`
+ORTP_CFLAGS=`pkg-config --cflags ortp`
 
 AM_CFLAGS=$(ORTP_CFLAGS) $(SPEEX_CFLAGS) $(STRICT_OPTIONS)
--- linphone-1.3.5/support/Makefile.am.wiget	2006-02-18 22:51:52.000000000 +0100
+++ linphone-1.3.5/support/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -13,4 +13,4 @@
 					gspawn.c gspawn.h \
 					gshell.c gshell.h
 
-INCLUDES=-I$(top_srcdir)/oRTP/include
\ Brak znaku nowej linii na końcu pliku
+INCLUDES=-I/usr/include/ortp
--- linphone-1.3.5/mediastreamer/Makefile.am.wiget	2006-03-28 16:36:26.000000000 +0200
+++ linphone-1.3.5/mediastreamer/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -77,9 +77,9 @@
 endif
 
 libmediastreamer_la_LIBADD= $(GLIB_LIBS) \
-							../gsmlib/libgsm.la \
-							../lpc10-1.5/liblpc10.la \
-							../oRTP/src/libortp.la \
+							-lgsm \
+							-llpc10 \
+							-lortp \
 							$(JACK_LIBS)\
 							$(SAMPLERATE_LIBS)\
 							$(SUPPORTLIB) \
@@ -163,9 +163,7 @@
 
 INCLUDES=  -I$(top_srcdir) \
 						-I$(top_srcdir)/mediastreamer \
-						-I$(top_srcdir)/oRTP/include \
-						-I$(top_srcdir)/gsmlib   \
-						-I$(top_srcdir)/lpc10-1.5    \
+						-I/usr/include/ortp \
 						$(SPEEX_CFLAGS) \
 						$(TRUESPEECH_INCLUDES) 
 
--- linphone-1.3.5/media_api/Makefile.am.wiget	2002-09-18 20:53:34.000000000 +0200
+++ linphone-1.3.5/media_api/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -22,11 +22,7 @@
 DEFS=@DEFS@ @SOUNDDEFS@ -DDEBUG -DG_LOG_DOMAIN=\"MediaApi\" 
 
 INCLUDES=-I$(top_srcdir)/mediastreamer \
-		-I$(top_srcdir)/speex \
-		-I$(top_srcdir)/gsmlib \
-		-I$(top_srcdir)/oRTP \
-		-I$(top_srcdir)/oRTP/src \
-		-I$(top_srcdir)/lpc10-1.5 \
-		-I$(top_srcdir)/ffmpeg 
+		-I/usr/include/ortp \
+		-I/usr/include/ffmpeg 
 
 
--- linphone-1.3.5/coreapi/Makefile.am.wiget	2006-02-14 17:37:35.000000000 +0100
+++ linphone-1.3.5/coreapi/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -8,9 +8,7 @@
 INCLUDES = \
 	-I$(top_srcdir)\
 	-I$(top_srcdir)/exosip\
-	-I$(top_srcdir)/oRTP/include \
-	-I$(top_builddir)/oRTP/ \
-	-I$(top_srcdir)/oRTP/ \
+	-I/usr/include/ortp \
 	-I$(top_srcdir)/mediastreamer \
 	$(SPEEX_CFLAGS) \
 	-I$(osip_prefix)/include/ 
--- linphone-1.3.5/console/Makefile.am.wiget	2006-01-29 13:11:50.000000000 +0100
+++ linphone-1.3.5/console/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -5,9 +5,7 @@
 	-I$(top_srcdir)/coreapi\
 	-I$(top_srcdir)/osipua/\
 	-I$(top_srcdir)/osipua/src \
-	-I$(top_srcdir)/oRTP/include \
-	-I$(top_builddir)/oRTP/ \
-	-I$(top_srcdir)/oRTP/ \
+	-I/usr/include/ortp \
 	-I$(top_srcdir)/mediastreamer \
 	$(SPEEX_CFLAGS)\
 	-I$(osip_prefix)/include \
@@ -32,7 +30,7 @@
 #in order to workaround a arm-linux cross-compilation libtool bug
 #we need to explicit again the dependencies of the executables:
 if ARMBUILD
-AM_LDFLAGS= 	$(top_builddir)/oRTP/src/libortp.la \
+AM_LDFLAGS= 	-lortp \
 				$(SPEEX_LIBS) \
 				$(OSIP_LIBS)
 endif
--- linphone-1.3.5/gnome/Makefile.am.wiget	2006-01-23 17:53:48.000000000 +0100
+++ linphone-1.3.5/gnome/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -17,10 +17,8 @@
 INCLUDES = \
 	-I$(top_srcdir)\
 	-I$(top_srcdir)/intl \
-	-I$(top_srcdir)/oRTP/src \
-	-I$(top_srcdir)/oRTP/ \
+	-I/usr/include/ortp \
 	-I$(top_srcdir)/mediastreamer \
-	-I$(top_srcdir)/speex/libspeex \
 	-I$(top_srcdir)/coreapi \
 	-I$(top_srcdir)/osipua/ \
 	-I$(top_srcdir)/osipua/src \
--- linphone-1.3.5/Makefile.am.wiget	2006-04-04 10:18:47.000000000 +0200
+++ linphone-1.3.5/Makefile.am	2006-05-04 23:43:41.000000000 +0200
@@ -5,7 +5,7 @@
 
 
 
-SUBDIRS = m4 support pixmaps po ipkg oRTP gsmlib lpc10-1.5 \
+SUBDIRS = m4 support pixmaps po ipkg \
                 $(WIN32ACM_DIR) mediastreamer mediastreamer2\
 		media_api exosip coreapi console gnome share developer-docs
 
