--- linphone-desktop-4.4.11/linphone-app/cmake_builder/linphone_package/CMakeLists.txt.orig	2022-11-03 08:37:39.000000000 +0100
+++ linphone-desktop-4.4.11/linphone-app/cmake_builder/linphone_package/CMakeLists.txt	2023-04-19 22:47:47.690707946 +0200
@@ -231,7 +231,9 @@ else()# Not Windows and Apple
 		execute_process(COMMAND install_name_tool -rpath "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}" "$ORIGIN/../lib" "${LIBRARY}")
 		execute_process(COMMAND install_name_tool -addrpath "$ORIGIN/../lib64" "${LIBRARY}")
 	endforeach ()
+	if (EXISTS "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_BINDIR}")
 	install(DIRECTORY "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_BINDIR}/" DESTINATION "${CMAKE_INSTALL_BINDIR}" USE_SOURCE_PERMISSIONS)
+	endif()
 #Just in case. This is useless because we have to use CMAKE_INSTALL_LIBDIR
 	if( EXISTS "${LINPHONE_OUTPUT_DIR}/lib/")
 		file(GLOB SHARED_LIBRARIES "${LINPHONE_OUTPUT_DIR}/lib/*.so*")
@@ -257,6 +259,7 @@ else()# Not Windows and Apple
 		endif()
 		install(FILES ${SHARED_LIBRARIES} DESTINATION "lib64")
 	endif()
+	if (EXISTS "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_DATAROOTDIR}")
 	install(DIRECTORY "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}" USE_SOURCE_PERMISSIONS)
 	install(DIRECTORY "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}" USE_SOURCE_PERMISSIONS
 			PATTERN "linphone" EXCLUDE
@@ -268,8 +271,11 @@ else()# Not Windows and Apple
 	if(ENABLE_BUILD_VERBOSE)
 		message("INSTALLATION : ${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/" )
 	endif()
+	endif()
+	if (EXISTS "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}")
 	file(GLOB PLUGINS_FILES "${LINPHONE_OUTPUT_DIR}/${CMAKE_INSTALL_LIBDIR}/mediastreamer/plugins/*")
 	install(FILES ${PLUGINS_FILES} DESTINATION "plugins/mediastreamer/" )
+	endif()
 # Install desktop/icon files.
 	configure_file("${CMAKE_CURRENT_SOURCE_DIR}/../../assets/linphone.desktop.cmake" "${CMAKE_CURRENT_BINARY_DIR}/../../${EXECUTABLE_NAME}.desktop" @ONLY)	
 	install(FILES "${CMAKE_CURRENT_BINARY_DIR}/../../${EXECUTABLE_NAME}.desktop" DESTINATION "${CMAKE_INSTALL_DATADIR}/applications" PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
