--- linphone-1.2.0/share/linphone.desktop~	2006-01-23 08:43:33.000000000 +0100
+++ linphone-1.2.0/share/linphone.desktop	2006-01-23 08:43:33.000000000 +0100
@@ -1,10 +1,12 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Linphone
 Comment=Linphone is a web-phone
-Comment[fr]=Linphone est un web-phone.
-Comment[de]=Linphone ist ein web-phone.
+Comment[fr]=Linphone est un web-phone
+Comment[de]=Linphone ist ein web-phone
+Comment[pl]=Telefon internetowy Linphone
 Type=Application
 Exec=linphone
 Icon=linphone/linphone2.png
-Terminal=0
+Terminal=false
 Categories=Application;Network;
--- linphone-1.2.0/share/linphone_applet.desktop~	2006-01-23 08:43:33.000000000 +0100
+++ linphone-1.2.0/share/linphone_applet.desktop	2006-01-23 08:43:33.000000000 +0100
@@ -1,9 +1,11 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Linphone Applet
 Comment=Linphone is a web-phone
-Comment[fr]=Linphone est un web-phone.
-Comment[de]=Linphone ist ein web-phone.
+Comment[fr]=Linphone est un web-phone
+Comment[de]=Linphone ist ein web-phone
+Comment[pl]=Telefon internetowy Linphone
 Type=Application
 Exec=linphone --applet --activate-goad-server=linphone_applet
 Icon=linphone/linphone2.png
-Terminal=0
\ No newline at end of file
+Terminal=false
